boa_ast/src/function/class.rs

```diff
@@ -90,21 +90,23 @@ impl ToIndentedString for Class {
         if self.elements.is_empty() && self.constructor().is_none() {
             return format!(
                 "class {class_name}{} {{}}",
-                if let Some(sup) = &self.super_ref {
-                    format!(" extends {}", sup.to_interned_string(interner))
-                } else {
-                    String::new()
-                }
+                self.super_ref
+                    .as_ref()
+                    .map_or_else(String::new, |sup| format!(
```

Comment:
This could be a `map_or_default`

Reply:
`map_or_default` does not exist for `Option`.